releases/nightly-build/compose-files/docker-compose-nexus-arm64.yml

```
@@ -166,7 +166,7 @@ services:
         - 'POSTGRES_USER=kong'
 
   kong-migrations:
-    image: kong:1.3.0-ubuntu
+    image: kong:2.0.0-ubuntu
```

IMO set sensible defaults and let environment variables override. 

Would benefit other repositories such as `blackbox-testing` while keeping backwards compatibility
```suggestion
    image: kong:${KONG_VERSION:-2.0.1}-ubuntu
```